src/components/Comment/CommentsList.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useCallback, useContext } from 'react';
+import { Comment } from '../../types/Comment';
+import { DispatchContext } from '../../reducer/store';
+import { commentService } from '../../services/comment.service';
+
+type Props = {
+  comments: Comment[]
+};
+
+export const CommentsList: React.FC<Props> = ({ comments }) => {
+  const dispatch = useContext(DispatchContext);
+
+  const onDeleteButton = useCallback((id: number) => () => {
+    const { removeComment } = commentService(dispatch, comments);
+
+    removeComment(id);
+  }, [comments]);
+
+  return (
+    <>
+      <p className="title is-4">Comments:</p>
+
+      {comments.map(comment => {
```

Comment:
You are directly returning a JSX element from the map function. While this works, it can lead to more difficult debugging due to the lack of a return statement. Consider wrapping the returned JSX in a return statement for clarity.